appLegacyFragment/src/main/java/com/hamdan/forzenbook/legacy/fragment/app/view/NavigatorImpl.kt

```diff
@@ -10,4 +10,7 @@ class NavigatorImpl : Navigator {
 
     override fun navigateToCreateAccount(context: Context) {
     }
+
+    override fun navigateToPost(context: Context) {
```

Comment:
TODOs for these? Why empty?

Reply:
theres a comment above the class:
"Need an empty impl of Navigator for dagger to be happy"
Should we try to seperate where the navigators are created then?